### DISCUSSION TITLE
How to connect to existing meeting using JWT app?

### DISCUSSION THREAD

**Discussion by fnc12** [Jericho/ZoomNet#171] [General]
This library seems very nice but I cannot find any example of connecting to a meeting. Thanks in advance

**Comment by Jericho** [comment]
Do you mean interactively join a meeting? Say, in a desktop app for example? If that's what you mean, I'm afraid ZoomNet is not the tool you are looking for. You need to investigate one of [Zoom's SDKs](https://marketplace.zoom.us/docs/sdk/). Their "Video SDK" or their "Meeting SDK" is probably what you are looking for.

Let me know if I misunderstood your question.

For the record: ZoomNet is a library that helps you read and write to/from Zoom's API resources and mirror some of the most popular features available in Zoom Web Portal such as creating a new meeting, creating, adding and removing users, viewing reports and dashboards on various usage, and so on.
